src/components/BulkEditList/BulkEditListResult/BulkEditInApp/BulkEditInApp.js

```diff
@@ -9,18 +9,21 @@ import { Headline,
   Row,
   Accordion,
   Select,
-  RepeatableField } from '@folio/stripes/components';
+  RepeatableField,
+  Datepicker } from '@folio/stripes/components';
```

Comment:
to avoid extra diffs in PRs people follow next codestyle with trailing comma

```
{
  Smth1,
  Smth2,
} from '' 
```

but it's your module, up to you @uladislausamets

Reply:
I agree with that point. I fixed it already. Thanks a lot!